### ISSUE TITLE
version 0.10 reports error for every single docstring

### ISSUE THREAD

**Issue by Ichimonji10** [zheller/flake8-quotes#55]
flake8-quotes version 0.10 reports error Q000 for every single Python docstring everywhere. For example:

```bash
flake8 - <<EOF
> """Print hello world."""
> print('Hello, world!')
> EOF
stdin:1:1: Q000 Remove bad quotes.
```

**Comment by cas--** [comment]
I just found this too with failing tests with push to ci...

#52 pull-request that instigated it. However I would like to point out that PEP8 clearly states:

> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257 . 

So that would suggest this addition was never needed though it looks like it will be fixed to not be default soon: #54

**Comment by fxfitz** [comment]
We're hitting this too. Any idea how fast we can get a bug fix released?

**Comment by twolfson** [comment]
Hey folks, sorry about that. I failed to realize that `0.10.0` wasn't backwards compatible during my review. We've released a patched `0.10.1` which uses `"""` as the default behavior which should support everyone's behavior

**Comment by fxfitz** [comment]
No worries! Thanks for the quick fix! :-D
